app/views/users/phone_setup/index.html.erb

```diff
@@ -49,3 +50,14 @@
 <% end %>
 
 <%= render 'shared/cancel_or_back_to_options' %>
+<% if FeatureManagement.phone_recaptcha_enabled? %>
+  <p class="margin-top-3">
```

Comment:
Couple nit-picky spacing observations: The top margin should be a bit larger based on the designs†, and a paragraph has a default bottom margin we'd want to remove.

```suggestion
  <p class="margin-top-4 margin-bottom-0">
```

† The design has 40px (5 units) between the paragraph and the "Choose another option" link, but I think the Figma spacing is a bit misleading since it doesn't have an accurate line-height on the link, so I suggested 4 instead as a compromise.